File: PRB01.cpp
// https://www.codechef.com/problems/PRB01
#include <bits/stdc++.h>
using namespace std;
void primal(int n)
{
    bool flag = 0;
    for (int i = 2; i < n / 2; i++)
    {
        if (n % i == 0)
        {
            printf("no\n");
            flag = 1;
            break;
        }
    }
    if (flag == 0)
    {
        printf("yes\n");
    }
}
int main()
{
    int t, n;
    cin >> t;
    while (t--)
    {
        cin >> n;
        primal(n);
    }
    return 0;
}

File: PRB01.py
# https://www.codechef.com/problems/PRB01
import math


def primal(n):
    if(n < 2):
        return "no"
    for x in range(2, int(math.sqrt(n))+1):
        if(n % x == 0):
            return "no"
    return "yes"


if __name__ == "__main__":
    t = int(input())
    while(t):
        n = int(input())
        print(primal(n))
        t -= 1
